### PR TITLE
cart management

### PR DESCRIPTION
**Pull request by prakashsakari**
This PR has the following
1.  Add/Remove items to/from Cart
2. Increase quantity of items that increases the price
3. Move items from the cart to the wishlist

cart

### CODE REVIEW

File: mockbee-test/ecom-store/src/components/FinalPrice/FinalPrice.jsx [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+  return (
+    <div className="total-price padding-all-16 align-self">
+      <h3 className="cart-title">Price Details</h3>
+      <div className="price-distribution d-flex direction-column gap">
+        <div className="items-purchased d-flex align-center">
+          <p>Price ({cart.length} items)</p>
+          <p className="mg-left">Rs. {totalItemPrice}</p>
+        </div>
+        <div className="discount-rate d-flex align-center">
+          <p>Dsicount</p>
```
Comment (Aditi0505) [inline mockbee-test/ecom-store/src/components/FinalPrice/FinalPrice.jsx]: Typo. Please correct the spelling here.

File: mockbee-test/ecom-store/src/pages/Cart.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
+import { ProductCardHorizontal, FinalPrice, Navbar } from "../components";
+import { useCart } from "../context/cart-context";
+import { Link } from "react-router-dom";
+export const Cart = () => {
+  const {
+    cartState: { cart }
+  } = useCart();
+  console.log(cart);
```
Comment (Aditi0505) [inline mockbee-test/ecom-store/src/pages/Cart.js]: Please remove console.log

**Review by Aditi0505 — APPROVED** [review]
Left few comments. Rest looks good to me.
1. Good folder structure
2. Clean and readable code
3. Good naming convention followed

**Review by surbhi84 — APPROVED** [review]
code is clean and easily readable,
making a separate folder for reducer is a good call.
(Just a suggestion) you can also try to implement a quantity increase if the user clicks on the add to cart button again.

File: mockbee-test/ecom-store/src/components/FinalPrice/FinalPrice.jsx (hunk 1 above)
Comment (prakashsakari) [reply mockbee-test/ecom-store/src/components/FinalPrice/FinalPrice.jsx]: thanks, updated

File: mockbee-test/ecom-store/src/pages/Cart.js (hunk 2 above)
Comment (prakashsakari) [reply mockbee-test/ecom-store/src/pages/Cart.js]: thanks, updated

**Pull request merged**